fixtures/fixtures.ts

```diff
@@ -0,0 +1,23 @@
+import { test as base } from '@playwright/test';
+import { InventoryPage } from '../pages/inventoryPage';
+import { LoginPage } from '../pages/loginPage';
+import { User } from '../user-data/users';
+
+export type TestFixtures = {
+    loginPage: LoginPage;
+    inventoryPage: InventoryPage;
+    currentUser: User
+  };
+  
+export const test = base.extend<TestFixtures>({
+  currentUser: {} as User,
+  loginPage: async ({ page }, use) => {
+    const loginPage = new LoginPage(page);
```

Comment:
You could introduce static `open` method that will perform navigation and return instance of the page
```typescript
export class LoginPage  {
    static async open(page: Page) {
      await page.goto('');
      return new LoginPage(page);
 }

constructor(page: Page) {}
}
```